### DISCUSSION TITLE
Pre-Trained Models

### DISCUSSION THREAD

**Discussion by MrSaper112** [vladmandic/face-api#107] [Q&A]
I was trying to apply custom trained model but every time i got error. Is It possible?

**Comment by vladmandic** [comment]
what do you mean custom trained? what is the model you've used? how was it trained?

**Reply by MrSaper112** [reply]
Ok, sou I need to apply more accurate trained model ( folder weights) (in tensorflow), converted to .bin and .JSON, Is it possible in your API?

**Comment by vladmandic** [comment]
you need to say what are you even trying to do here? you cannot just use any model and combine it with a library of your choice such as face-api. face-api was designed to work with a specific model. unless your model architecture is identical, it will never work.

**Reply by MrSaper112** [reply]
Ok ,Thanks
